packages/react-components/index.js

```diff
@@ -38,6 +38,8 @@ import Telephone, {
 import TextArea from './TextArea';
 import TextInput from './TextInput';
 
+import './i18n-setup';
```

Comment:
And this import initializes the `i18next` dependency with our language detection and translation files, so when components import the dependency it's ready to use.